### PR TITLE
feat: add credits menu.

### PR DESCRIPTION
**Pull request by zicklag**

### CODE REVIEW

**Comment by erlend-sh** [comment]
You can move yourself and @zTecna up to Founding Team. We’ll move the ‘alumni’ down to Contributors:

```
- Fedor Logachev [@not-fl3](https://github.com/not-fl3) - Technical Director (former: v0.0-v0.2)
- Ole A. Sjo Fasting [@olefasting](https://github.com/olefasting) - Technical Director (former: v0.3-v0.4)
- Orlando Herrera [@_pixelfrog](https://twitter.com/_pixelfrog) - Art Director (former: v0.0-v0.3)
```

File: CREDITS.md [hunk 1]
```diff
@@ -1,42 +1,62 @@
-# Forerunners
+# Credits
```
Comment (zicklag) [inline CREDITS.md:1]: @erlend-sh I'd like to get your review on the updated credits. We can change this any time so it doesn't need to block the PR if you want to get to this later.

I added all the Git commiters that weren't previously in the list ( we'll probably want to automate this later ), and now the game will parse the Credits file to render the in-game credits so we don't have to maintain two lists:

image

I wasn't sure if there's more criteria for ending up in the credits or not.

Comment (zicklag) [reply CREDITS.md:1]: Ah, just saw your comment. 👍

Comment (erlend-sh) [reply CREDITS.md:1]: That’s excellent 👌 I think the only ones missing are some artists, so I’ll cover that in my second pass.

**Review by erlend-sh — APPROVED** [review]

**Pull request merged**